src/component/Approval.vue

```diff
@@ -88,7 +89,7 @@ export default {
         &__input, &__drop-down  {
             margin: 0 5px;
             border-radius: 5px;
-            border: 1px solid rgb(204, 204, 204);
+            border: 1px solid $shadow;
```

Comment:
What kind of color is `shadow`? In accordance to our naming system how about `input__grey` or something else?
Honestly, After reading this line, I'm starting to doubt if our naming system can be considered as concise